src/content/docs/reference/cli.mdx

```diff
@@ -4,9 +4,23 @@ description: A reference of supported operations using the Daytona CLI.
 sidebar:
   label: Daytona CLI Reference
 ---
+import Aside from "@components/Aside.astro";
 
-The `daytona` CLI provides access to all of Daytona's core features.
-You can access help for each command using the `--help`/`-h` flag.
+The `daytona` command-line tool provides access to Daytona's core features.
+The tool acts as both an interface to a running Daytona Server, and as an interface to start and stop the server.
```

Comment:
@osslate i feel this line is unnecessary and would remove as below you mention it also manages datyona server

Reply:
Resolved in [eeee47f](https://github.com/daytonaio/docs/pull/60/commits/eeee47fead5a67b524497678f45a5dffe8a3b578)